arch/arm64/boot/dts/intel/socfpga_agilex_bittware.dts

```diff
@@ -41,6 +41,34 @@
 		/* We expect the bootloader to fill in the reg */
 		reg = <0 0 0 0>;
 	};
+
```

Comment:
Ashish,  Just FYI, Dushmanta is also enabling CMA using same mechanism.  Please check and ensure that there is no conflicts.  Also,  please note  take a note of issues highlighted by NXP on CMA - https://community.nxp.com/t5/i-MX-Processors-Knowledge-Base/How-to-get-rid-of-CMA/ta-p/1123287

Reply:
Will do, I looked up this description and we can revisit after FPGA demo.